dapps/assets/src/components/DataContainer.ts

```diff
@@ -47,12 +49,41 @@ export default class DataContainerComponent extends mixins(EvanComponent) {
   })
   type: string;
 
+  /**
+   * Mapping of account addresses to name / alias resolving promises
+   */
+  aliasMapping: { [ address: string ]: Promise<string> } = { };
+
   mounted(): void {
     this.searchTerm = this.$route.params.query || '';
 
     this.initialQuery(this.searchTerm);
   }
 
+  /**
+   * Check the search results, if the name / alias of an owner can be loaded
+   */
+  async ensureOwnerNames(searchResults: DigitalTwin[]): Promise<void> {
+    const runtime = this.getRuntime();
+
+    await Promise.all(searchResults.map(async (result): Promise<void> => {
+      if (result.owner && runtime.web3.utils.isAddress(result.owner)) {
+        if (!this.aliasMapping[result.owner]) {
+          this.aliasMapping[result.owner] = bccUtils.getUserAlias(
+            new bcc.Profile({
+              accountId: runtime.activeAccount,
+              profileOwner: result.owner,
+              ...(runtime as bcc.ProfileOptions),
+            }),
+          );
+        }
+
+        // eslint-disable-next-line no-param-reassign
+        result.owner = await this.aliasMapping[result.owner];
```

Comment:
param reassign is by a reason discouraged from airbnb ;) 
maybe you can just return map of owners assign it from the function. Better function name would than also be getOwnerMapping e.g. ...

Reply:
Hmmm... It's just for using it twice. When i rework it into the getOwnerMapping function, i need to loop over this one, to apply it to the original one. In the context of this component, it makes totally sense, to just use this function using param reassign.